src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/VedtaksperiodeService.kt

```diff
@@ -56,10 +57,18 @@ class VedtaksperiodeService(
         return forrigeVedtaksperioder.avkortFraOgMed(behandling.revurderFra.minusDays(1))
     }
 
-    fun detFinnesVedtaksperioder(
-        behandlingId: BehandlingId,
-        forrigeBehandlingId: BehandlingId?,
-    ) = finnVedtaksperioder(behandlingId)?.isNotEmpty() == true || finnVedtaksperioder(forrigeBehandlingId)?.isNotEmpty() == true
+    fun detFinnesVedtaksperioderPåForrigeBehandling(saksbehandling: Saksbehandling): Boolean {
+        if (saksbehandling.type == BehandlingType.FØRSTEGANGSBEHANDLING ||
+            finnesTidligereIverksatteVedtak(saksbehandling)
+        ) {
+            return false
+        }
+
+        return finnVedtaksperioder(saksbehandling.forrigeBehandlingId)?.isNotEmpty() == true
+    }
+
+    private fun finnesTidligereIverksatteVedtak(saksbehandling: Saksbehandling) =
```

Comment:
Hvis denne fortsatt skal brukes
```suggestion
    private fun finnesIkkeTidligereIverksatteVedtak(saksbehandling: Saksbehandling) =
```